frontend/src/Tests/HowToAddLocation.test.tsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import HowToAddLocation from '../components/HowToAddLocation/HowToAddLocation';
+
+describe('Tests', () => {
```

Comment:
We can use a lot of test , maybe we need to use more width description of test block , for example:
describe('Big Popup tests ', () => {
  describe('Rendering block', () => {
    ('test something 1', () => { ... });
    ('test something 2', () =>{ ... });
   ('test something 3', () =>{ ... });
  });
});
describe('Personal profile tests ', () => {
  describe('Rendering block', () => {
    ('test something 1', () => { ... });
    ('test something 2', () =>{ ... });
    ('test something 3', () =>{ ... });
  });
});

Reply:
ok